Respect environment CPPFLAGS when we run ./configure so we can inject
random -D things without having to set CFLAGS/ASFLAGS

This patch was taken from gentoo.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 Makeconfig     |    1 +
 config.make.in |    1 +
 2 files changed, 2 insertions(+)

Index: glibc-2.13/Makeconfig
===================================================================
--- glibc-2.13.orig/Makeconfig
+++ glibc-2.13/Makeconfig
@@ -697,6 +697,7 @@ CPPFLAGS = $($(subdir)-CPPFLAGS) $(+incl
 	   $(foreach lib,$(libof-$(basename $(@F))) \
 			 $(libof-$(<F)) $(libof-$(@F)),$(CPPFLAGS-$(lib))) \
 	   $(CPPFLAGS-$(<F)) $(CPPFLAGS-$(@F)) $(CPPFLAGS-$(basename $(@F)))
+CPPFLAGS += $(CPPFLAGS-config)
 override CFLAGS	= -std=gnu99 $(gnu89-inline-CFLAGS) \
 		  $(filter-out %frame-pointer,$(+cflags)) $(+gccwarn-c) \
 		  $(sysdep-CFLAGS) $(CFLAGS-$(suffix $@)) $(CFLAGS-$(<F)) \
Index: glibc-2.13/config.make.in
===================================================================
--- glibc-2.13.orig/config.make.in
+++ glibc-2.13/config.make.in
@@ -107,6 +107,7 @@ CC = @CC@
 CXX = @CXX@
 BUILD_CC = @BUILD_CC@
 CFLAGS = @CFLAGS@
+CPPFLAGS-config = @CPPFLAGS@
 ASFLAGS-config = @ASFLAGS_config@
 AR = @AR@
 RANLIB = @RANLIB@
